playbook-website/config/webpack/environment.js

```diff
@@ -2,28 +2,56 @@ const path = require('path')
 
 const { environment } = require('@rails/webpacker')
 
-environment.loaders.insert('javascript', {
-  test: /\.(js|jsx)$/,
+const babelLoader = environment.loaders.get('babel')
+babelLoader.test = /\.(js|jsx|ts|tsx)?(\.erb)?$/
+
+const nodeLoader = environment.loaders.get('nodeModules')
+nodeLoader.exclude = [
+  nodeLoader.exclude,
+  /node_modules\/@powerhome\/playbook-icons-react/
+]
+
+environment.loaders.prepend('ESM', {
+  test: /\.mjs$/,
   use: {
     loader: 'babel-loader',
     options: {
       cacheDirectory: true,
     },
   },
-  include: path.resolve(__dirname, '../../app'),
+  // include: /node_modules/,
+  type: 'javascript/auto',
```

Comment:
This is the correct type mentioned above https://github.com/powerhome/playbook/pull/3187/files#r1506723269